RecoTracker/CkfPattern/plugins/GroupedCkfTrajectoryBuilder.cc

```diff
@@ -361,7 +361,7 @@ GroupedCkfTrajectoryBuilder::groupedLimitedCandidates (const TrajectorySeed& see
       if ((int)newCand.size() > theMaxCand) {
 	//ShowCand()(newCand);
 
- 	sort( newCand.begin(), newCand.end(), GroupedTrajCandLess(theLostHitPenalty,theFoundHitBonus));
+ 	std::nth_element( newCand.begin(), newCand.begin()+theMaxCand, newCand.end(), GroupedTrajCandLess(theLostHitPenalty,theFoundHitBonus));
```

Comment:
Does the remaining code using the "newCand" care about the order of the trajectories (the part that's left from begin to +maxCand)?